pages/legacy/content_publish.py

```diff
@@ -10,7 +10,9 @@
 class ContentPublish(PrivatePage):
     _publish_form_locator = (By.CSS_SELECTOR, '#content div.documentContent div div form')
     _message_textarea_locator = (By.CSS_SELECTOR, 'textarea[name="message"]')
-    _submit_button_locator = (By.CSS_SELECTOR, 'input[type="submit"][name="form.button.publish"]')
+    _submit_button_locator = (By.CSS_SELECTOR, 'input[type="submit"][name="form.button.publish"], '
+                                               'input[type="submit"][name="form.button.submit"]')
+    _block_msg_locator = (By.CSS_SELECTOR, '#region-content > div > div > div > div > div')
```

Comment:
:( was there any way to get a less flaky selector using classes maybe (it's fine if this is the only way).

Reply:
I am afraid that's the only way :(